<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Coffee">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>290</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Coffee</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Tea">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Tea</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Tang">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>350</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Tang</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Reset">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>420</y>
     <width>87</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>82</width>
     <height>124</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pb_Coin10">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_Coin50">
      <property name="text">
       <string>50</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_Coin100">
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_Coin500">
      <property name="text">
       <string>500</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
